httpd/handler.go

```diff
@@ -18,13 +18,17 @@ import (
 	"time"
 
 	"code.google.com/p/go-uuid/uuid"
-
 	"github.com/bmizerany/pat"
 	"github.com/influxdb/influxdb"
 	"github.com/influxdb/influxdb/client"
 	"github.com/influxdb/influxdb/influxql"
 )
 
+const (
+	// With raw data queries, mappers will read up to this amount before sending results back to the engine
+	DefaultChunkSize = 10000
```

Comment:
should we note in the comment if this is `bytes` or `rows` so it's clear in the docs?